.github/workflows/reusable-api-e2e.yml

```diff
@@ -29,37 +29,13 @@ jobs:
           cp .npmrc-cloud .npmrc
 
       - uses: ./.github/actions/setup-project
+      - uses: ./.github/actions/start-localstack
```

Comment:
use LocalStack action

Reply:
Failing fast rule, would suggest running the linter before so we can quit earlier from here

Reply:
sure, will do that! :)